### ISSUE TITLE
Firefox print dialog

### ISSUE THREAD

**Issue by fghiggi** [stimulsoft/Samples-Dashboards-NET.Core-3.1-MVC-CSharp#1]
Running `Export_and_Print_Dashboard_from_Code` project and printing a PDF in `Firefox`  a print popup shows up while browser is rendering the report.

![firefox-print-dialog](https://user-images.githubusercontent.com/9020705/108897360-c5723b80-75f4-11eb-89a0-816b8fb8f17c.jpg)

Is this a bug or i can disable it?

**Comment by VladimirHot** [comment]
Hello,

PrintAsPdf() prints the report to the printer, ResponseAsPdf() loads the PDF without printing. Use what suits you best.

**Comment by fghiggi** [comment]
Using `PrintAsPdf` in chrome does not have the same behavior then firefox. 
This print popup only appears in firefox, if is this the correct behavior from `PrintAsPdf`, chrome has a bug.

**Comment by VladimirHot** [comment]
That's right, since some time Chrome has a bug - the print script that is embedded in the PDF file is not executed.
